repo2docker/buildpacks/conda/freeze.py

```diff
@@ -35,7 +41,7 @@
 yaml = YAML(typ="rt")
 
 
-def freeze(env_file, frozen_file):
+def freeze(env_file, frozen_file, containerruntime="docker", hostmountopts=""):
```

Comment:
Is the `hostmountopts=""` on purpose? Asking because the 'pythonic' thing is to use `None` as the empty default value instead of `[]` or `""`. That way `None` is the "I am not passing anything, please use the default" value and `""` is the "I (the caller) passed `""` on purpose, please use it and not the default" value.

For my education: what are host mount options?

Reply:
I chose `""` because the default value should be no additional options to maintain the existing behaviour. I can change it to `None`, and add another statement `if hostmountopts is None: hostmountopts = ""` if that's preferable?

Thinking about it maybe `mountopts` is a better name the `hostmountopts` since in theory you could pass other Docker mount options though I can't think why (for example readonly `ro` is one option but that's obviously not much use for this script :smile: ).

If you have SELinux enabled by default it prevent containers mounting arbitrary filesystem directories such as home directories. To give it permissions you have to relabel (set soem SELinux attributes) on the directory you want to mount, either manually, or by passing the `z` mount option to tell docker or podman to automatically relabel the directory.

Reply:
Ja dunno. The `if hostmounopts is None:` pattern is more typing but maybe also more  common" No strong feelings really.